### PR TITLE
Query update animals table

### PR DESCRIPTION
**Pull request by Sami-Ullah-Tufail**
### I have updated the following functions
d2 data 10
d2 data 9
d2 data 8
d2 data 7
D2 data 6
d2 data 5
d2 data 4
d2 data 3
d2 data  2
d2 data 1

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+-- DAY 2
+-- Update the animals table by setting the species column to unspecified.
+-- Verify that change was made. Then roll back the change
+-- Verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+TABLE animals;
+ROLLBACK;
```
Comment (aamirkhan2478) [inline queries.sql:25]: - [x] Please use the `TABLE animals` query after `COMIT` to verify that change was undone :slightly_smiling_face:

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,72 @@
...
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+-- Commit the transaction.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+TABLE animals;
+COMMIT;
```
Comment (aamirkhan2478) [inline queries.sql:34]: - [x] Please use the `TABLE animals` query after `COMIT` to verify that change was undone :slightly_smiling_face:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @Sami-Ullah-Tufail, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (OyePriscilla) [reply queries.sql:34]: - [ ]  In accordance to the project requirement, we are to verify if the changes persist after "`SET species`" command, to achieve this I think you can use the select query to check as suggested earlier. 🙏

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Sami-Ullah-Tufail 👏 👏 👏 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Implement sql query ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @Sami-Ullah-Tufail 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Implemented requested changes
-  ✅ Great work using the right Gitflow  

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**